conda/environments/all_cuda-122.yaml

```diff
@@ -7,9 +7,12 @@ channels:
 - conda-forge
 - nvidia
 dependencies:
+- clang-tools>=19.0
+- clangxx>=19.0
 - cmake>=3.24.0,!=3.30.0
 - cuda-cudart-dev
 - cuda-nvcc
+- cuda-toolkit
```

Comment:
Not sure I understand where these come from suddenly based on the `dependencies.yaml` change and `pre-commit` doesn't complain?  (I guess I am missing something obvious.)

Reply:
They appeared because I forgot to add the clang-tidy dependencies to the 'all' section of dependencies.yaml.

Reply:
Thanks, somehow I it didn't occur to me that it was a section :facepalm: not a package.